config/locales/notifications/en.yml

```diff
@@ -16,6 +16,12 @@ en:
     added_to_contributors_page:
       1: >
         You now appear on our Contributors page. Thank you for contributing to Exercism!
+
+    # Not actually used
```

Comment:
We need these here right now else things break, but at some point I'll refactor them out. We don't show them in the UI, because they're email-only-notifications.